### DISCUSSION TITLE
@iconify/utils@2.1.20+ slows down the vite dev server

### DISCUSSION THREAD

**Discussion by zam157** [unocss/unocss#3819] [Q&A] [answered]
My Vue project unocss version has been stuck at `v0.58.3`. If I upgrade to a newer version, the startup speed of my vite dev server will be significantly slow. It takes about 1-2 minutes to see the page fully displayed, while the normal startup speed before upgrading unocss is about 10-20 seconds

Here's my unocss config

```typescript
import { defineConfig, presetUno, transformerDirectives, presetIcons } from 'unocss';

export default defineConfig({
  presets: [
    presetUno(),
    presetIcons({
      prefix: ['', 'i-'],
      extraProperties: {
        display: 'inline-block',
        'vertical-align': 'middle',
      },
    }),
  ],
  transformers: [transformerDirectives()],
  content: {
    pipeline: {
      include: [/\.(vue|tsx|html)($|\?)/, 'src/router/**/*.ts', 'src/views/**/modules/*.ts'],
    },
  },
});
```

I found that after setting
```json
"resolutions": {
  "@iconify/utils": "2.1.19"
}
```
to `package.json` and upgrade unocss to the latest version, the startup speed will return to normal. So I think it may be caused by the update of the `@iconify/utils` version used in `@unocss/preset-icons`

Has anyone encountered this problem?

**Comment by userquin** [comment, marked as answer]
Dont use empty icon prefix.

**Reply by userquin** [reply]
I Will check changes made between those version (iconify utils)

**Reply by zam157** [reply]
Thanks!
